js/loopjsfunfuzz.py

```diff
@@ -59,12 +59,9 @@ def parseOpts(args):
     # lower = less time wasted in timeouts and in compareJIT testcases that are thrown away due to OOMs.
     options.timeout = int(args[0])
 
+    # FIXME: We should remove the use of args[1] globally since it is no longer used.
     options.knownPath = os.path.expanduser(args[1])
-    # FIXME: findIgnoreLists.py should probably check this automatically.
-    reposWithKnownLists = ['mozilla-central', 'mozilla-esr45', 'ionmonkey', 'jscore', 'v8']
```

Comment:
Yes. The known lists are no longer valid since the move to FuzzManager. Instead of treating this block of changes as "ESR-removal", it should be regarded as "known-list-support-removal".